src/components/home/WhereWeWork.tsx

```diff
@@ -0,0 +1,106 @@
+import Image from "next/image";
+import { Box, Card, Flex, Grid, Heading, Text } from "@radix-ui/themes";
+
+type BackgroundColor = `bg-[#${string}]`;
+
+import europeMap from "./maps/europe.svg";
+import middleEastMap from "./maps/middle-east.svg";
+import unitedStatesMap from "./maps/united-states.svg";
+
+const WhereWeWork = () => {
+  const mapKey: {
+    label: string;
+    color: BackgroundColor;
+  }[] = [
+    {
+      label: "Receiving Aid",
+      color: "bg-[#000080]",
+    },
+    {
+      label: "Sending Aid",
+      color: "bg-[#5FBCD3]",
+    },
+    {
+      label: "Both",
+      color: "bg-[#5FD38D]",
+    },
+  ];
+
+  const cardContent: {
+    map: React.SVGElementType;
```

Comment:
Since the maps are not just decorative, we should have alt text for them. This could be added here.